microcosm_eventsource/stores/rollup.py

```diff
@@ -251,3 +267,9 @@ def _to_model(self, aggregate, event, container, *args):
             },
             *args[len(keys):]
         )
+
+    def _is_polymorphic(self):
+        if hasattr(self.container_type, "__mapper_args__"):
```

Comment:
Does this work if we use single table inheritance (which is what I've tended to use)?

I wouldn't want to change working behavior somewhere that uses single table inheritance without reason and testing.

Reply:
The problem I'm trying to solve here will appear exclusively for joined table inheritance, but I expect the workaround to work in all cases (though slightly less performant because of extra `join` - in extreme cases we may override `_is_polymorphic` to always return `False`).